api.go

```diff
@@ -638,10 +639,11 @@ func StatusContainer(podID, containerID string) (ContainerStatus, error) {
 	for _, container := range pod.containers {
 		if container.id == containerID {
 			return ContainerStatus{
-				ID:     container.id,
-				State:  container.state,
-				PID:    container.process.Pid,
-				RootFs: container.config.RootFs,
+				ID:          container.id,
+				State:       container.state,
+				PID:         container.process.Pid,
+				RootFs:      container.config.RootFs,
+				Annotations: container.pod.config.Annotations,
```

Comment:
This will work in the 1container/pod OCI context, but is not generic enough.

The container status annotations map should carry the container config annotations, not the pod ones, in order to carry container specific annotations in a context where we have more than 1 container per pod.

That means adding an `Annotations` field to the `ContainerConfig` structure and copying it to `ContainerStatus`